src/client/app/translations/data.js

```diff
@@ -1266,33 +1265,33 @@ const localeData = {
 		"meter.minDate": "(need Spanish) Minimum Reading Date Check",
 		"meter.maxDate": "(need Spanish) Maximum Reading Date Check",
 		"meter.maxError": "(need Spanish) Maximum Number of Errors Check",
-		"meter.disableChecks": "(need Spanish) Disable Checks",
-		"meter.failed.to.create.meter": "(Need Spanish) Failed to create a meter with message: ",
-		"meter.failed.to.edit.meter": "(Need Spanish) Failed to edit meter with message: ",
-		"meter.hidden": "(Need Spanish) At least one meter is not visible to you",
+		"meter.disableChecks": "Desactivar checks",
```

Comment:
I think checks might be "chequear" or something like that. This is meant to be check as in a test.